response_operations_ui/templates/conversation-view/cv-handle-errors.html

```diff
@@ -0,0 +1,22 @@
+{% if form.errors %}
+    {% set error_count = form.errors | length %}
+    {% set plural_suffix = 's' if error_count > 1 else '' %}
+    {% set header_text = 'This page has' ~ error_count ~ 'error' ~ plural_suffix %}
```

Comment:
spaces after `has` and before `error` needed or else it'll read "This page has20errors"?